charts/scalardl-audit/values.yaml

```diff
@@ -149,6 +149,65 @@ auditor:
     # -- The secret key of an Auditor private key
     auditorPrivateKeySecretKey: private-key
 
+  # -- The auditor.properties is created based on the values of auditor.scalarAuditorConfiguration by default. If you want to customize auditor.properties, you can override this value with your auditor.properties.
+  # @default -- The minimum template of auditor.properties is set by default.
```

Comment:
Ditto.

Reply:
I updated it in 3f3699880f21ff9b399ccff625cab32959a67aee based on the following comment.
https://github.com/scalar-labs/helm-charts/pull/102#discussion_r902149312